protos/bottle/catalog/v1/addon.proto

```diff
@@ -6,6 +6,8 @@ message Addon {
   enum AddonCategory {
     ADDON_CATEGORY_UNSPECIFIED = 0;
     ADDON_CATEGORY_RUSH_ASSEMBLY = 1;
+    ADDON_CATEGORY_HANDLING = 2;
+    ADDON_CATEGORY_EWASTE = 3;
```

Comment:
Both of these seem more like fees than addons because they are not user selected. I'm not sure if we want to split them out early or just keep them as addons until we decide there is a data structure difference